src/pages/home/report/ReportActionCompose.js

```diff
@@ -163,6 +178,34 @@ class ReportActionCompose extends React.Component {
         }
     }
 
+    /**
+     * Show the ReportActionContextMenu modal popover.
+     *
+     * @param {Object} [event] - A press event.
+     */
+    showEmojiPicker(event) {
+        this.textInput.blur();
```

Comment:
Do we want to do blur the `textInput` on mobile since there's no search bar there anymore? Might not be worth fixing, but just thought I'd point out that it might not be necessary/desirable

Reply:
> Do we want to do blur the textInput on mobile since there's no search bar there anymore?

Yeah so this is blurring the main compose input not the search input. It's actually _only_ important on mobile. If we don't do this, when the emoji picker comes up and they try to tap an emoji the first tap blurs the main compose input and the second one will actually select the emoji which we don't want